### ISSUE TITLE
Where did you hear about inquire?

### ISSUE THREAD

**Issue by mikaelmello** [mikaelmello/inquire#128]
### Discussed in https://github.com/mikaelmello/inquire/discussions/127

<div type='discussions-op-text'>

<sup>Originally posted by **mikaelmello** April 16, 2023</sup>
I really like to see what people are building with this, from the first few users to the several hundreds of stars this repo has today, it makes me really glad that people use and like this library.

2021 was the last time I posted publicly about this (on Reddit), and 700+ stars laters I'm really curious to know how people hear about it. There were even a few sharp increases in stars of which I have no idea where they came from lol

So, where did you hear about this library?</div>

**Comment by CM-IV** [comment]
I love using this crate for my terminal projects.  I don't remember exactly, but I was looking for similar libraries for NodeJS/Deno for terminal prompts and I stumbled upon `inquire`.

I've made a terminal ERP application in Rust that keeps track of projects, services, clients, crypto payments, and invoices with this awesome crate.  I've also made an image processing application with it that uses the `image_compressor` and `rayon` crates to work with folders of JPG images.  I even used `inquire` to make a terminal program that help me manage my vegetable gardens in the yard.

**TLDR**
I think `inquire` is awesome, here's what I've used it for:

- A linear algebra calculator
- An ERP application for my business
- An inventory application for things around the house
- A program to help me in the garden
- A file encryption terminal application derived from a `Salvo` web API that I wrote

**Comment by rpigab** [comment]
Google search : "terminal selector with autocomplete in rust"
First result, didn't have to change keywords.
I'm glad this exists, it looks really nice, I haven't tried yet but I'm sure it will be useful to me :+1:

**Comment by davidepedranz** [comment]
Google search: `rust interactive cli` :-D

Very glad of having found this library: it is exceeding my expectations! I didn't think that building interactive CLI would have been so convenient.

**Comment by russellbanks** [comment]
[blessed.rs](https://blessed.rs)

**Comment by mikaelmello** [comment]
@russellbanks this is very cool, hadn't heard of that before!

**Comment by Swivelgames** [comment]
A ChatGPT [conversation](https://chat.openai.com/share/bdbfdce3-3c12-4399-8b2c-b47a1193e828) I had pointed me to `inquire`. Which is great, because I was having trouble finding a good library. This seems to be exactly what I was looking for :tada:

**Comment by gs-101** [comment]
It's used by a CLI app [pay-respects](https://codeberg.org/iff/pay-respects), which I [packaged for Guix](https://github.com/gs-101/selected-guix-works/commit/38af58659d0ce75c10d14b7300c387c9a0823b).

I wanted Emacsier keybinds (n/p to be exact), so I let rust-analyzer take me to the place where was this implemented, and then ended up here.

I'll look into the codebase to see if I can do anythhing (should be the same code for Vim but with 'n/p').

**Comment by jarjk** [comment]
the awesome [lib.rs](https://lib.rs)
